CHANGELOG.md

```diff
@@ -2519,8 +2519,7 @@ With new plugins, many improvements and bug fixes, this release comes with break
 
 ### Breaking changes
 
-Several breaking changes are introduced. You will have to slightly change your configuration file and a migration script will take care of updating your database cluster. **Please follow the instructions in [UPDATE.md](/UPDATE.md#update-to-kong-050) for an update without downtime**.
```

Comment:
This is a link we still want, the proper link is now `[UPGRADE.md](/UPGRADE.md#update-to-kong-050)`

Reply:
Thank you for the update, I have updated the above link on my repository for Kong. Please validate and revert if anything, also let me know in case I need to submit a new PR for the same! Thank you.